### PR TITLE
docs(NET-1125): add download links for RAC mobile

### PR DESCRIPTION
**Pull request by Aceix**

### CODE REVIEW

File: pro/rac.rst
Comment (abhishek9686) [inline pro/rac.rst:1]: can we display qr code instead of links?
that will be easier for user to just use their mobile phone to scan and visit the app

**Review by abhishek9686 — CHANGES_REQUESTED** [review]

File: pro/rac.rst
Comment (Aceix) [reply pro/rac.rst:1]: image

File: pro/rac.rst [hunk 1]
```diff
@@ -10,15 +10,42 @@ RAC is mostly suited for offsite machines that need access to a Netmaker network
 Download/Installation
 ***********************
 
-You can download the latest version of RAC from the Netmaker file server:
+**For Mac**, You can download the mac installer from the fileserver link above and run it. Because the app needs to be run as root, open a teminal and enter the command ``sudo /Applications/NetmakerRemoteClient.app/Contents/MacOS/remote-client``.
 
-.. code-block::
+**For Windows**, you can download the remote-access-client_86.msi installer and run it to install on your windows machine. The app will need to be run as administrator. Right click on the desktop icon after installation and click on ``run as administrator``.
```
Comment (theguy951357) [inline pro/rac.rst:15]: include both msi and bundle exe file link. admin instructions not needed.

File: pro/rac.rst [hunk 2]
```diff
@@ -10,15 +10,42 @@ RAC is mostly suited for offsite machines that need access to a Netmaker network
 Download/Installation
 ***********************
 
-You can download the latest version of RAC from the Netmaker file server:
+**For Mac**, You can download the mac installer from the fileserver link above and run it. Because the app needs to be run as root, open a teminal and enter the command ``sudo /Applications/NetmakerRemoteClient.app/Contents/MacOS/remote-client``.
```
Comment (theguy951357) [inline pro/rac.rst:13]: no longer need to run as sudo. installer is fixed.

**Review by abhishek9686 — APPROVED** [review]

**Pull request merged**